extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -125,6 +140,37 @@ class AmpStickyAd extends AMP.BaseElement {
       });
     }
   }
+
+  /**
+   * The function that add a close button to sticky ad
+   * @private
+   */
+  addCloseButton_() {
+    if (this.closeButton_ && this.boundOnClick_) {
+      return;
+    }
+    const closeButton = this.getWin().document.createElement('div');
+    closeButton.setAttribute('class', '-amp-sticky-ad-close-button');
+    closeButton./*OK*/innerHTML = '&#10006';
```

Comment:
You just need to set it via JS unicode, e.g. `textContent = '\u####'` - whichever hex code happens to be.